.github/actions/download-manifests/action.yml

```diff
@@ -83,6 +83,11 @@ runs:
       with:
         name: tensorflow-notebook-x86_64-history_line
         path: ${{ inputs.histLineDir }}
+    - name: Download artifact 📥
+      uses: actions/download-artifact@v3
+      with:
+        name: pytorch-notebook-x86_64-history_line
+        path: ${{ inputs.histLineDir }}
```

Comment:
You need to download aarch64 artifact as well.

Reply:
Am I missing something , or are the following steps missing then as well?

```
    - name: Download artifact 📥
      uses: actions/download-artifact@v3
      with:
        name: tensorflow-notebook-aarch64-manifest
        path: ${{ inputs.manifestDir }}
```
and
```
    - name: Download artifact 📥
      uses: actions/download-artifact@v3
      with:
        name: tensorflow-notebook-aarch64-history_line
        path: ${{ inputs.histLineDir }}
```

Reply:
Thanks, you're right, it's a bug.
I merged a fix to the main branch.